herald.py

```diff
@@ -129,26 +131,41 @@ async def runHerald(ctx, args):
         return
 
     pickle.dump(heraldDict, open("herald/heraldUsers.p", "wb"))
+    return
```

Comment:
This isn't necessary since it's already the end of the function, right?

Reply:
You are correct, I just added it to be more explicit and for consistency with the rest of the blocks. It will make no difference in the program.